.github/workflows/test-currency-transactions.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          sbt -Dversion.override=99.99.99 --error clean compile sdk/publishM2 shared/publishM2 kernel/publishM2 keytool/publishM2 nodeShared/publishM2 currencyL0/publishM2 currencyL1/publishM2 dagL1/publishM2
```

Comment:
publication is skipped for all other modules but sdk, so these additional commands wouldn't create any artifacts.